js/app.js

```diff
@@ -0,0 +1,15 @@
+const getElement = (selector) => {
+    const element = document.querySelector(selector)
+  
+    if (element) return element
+    throw Error(
+      `Please double check your class names, there is no ${selector} class`
+    )
+  }
+  
+  const links = getElement('.nav-links')
+  const navBtnDOM = getElement('.nav-btn')
```

Comment:
@matycarolina it's a bad practice to use css classes for js, instead, you could create your own classes with a `js-` prefix, or even better use aria-attr to look for the elements you want